src/web-mercator-viewport.js

```diff
@@ -59,12 +61,12 @@ export default class WebMercatorViewport extends Viewport {
     width = width || 1;
     height = height || 1;
 
-    const scale = Math.pow(2, zoom);
+    const scale = zoomToScale(zoom);
     // Altitude - prevent division by 0
     // TODO - just throw an Error instead?
     altitude = Math.max(0.75, altitude);
 
-    const center = projectFlat([longitude, latitude], scale);
+    const center = getWorldPosition({longitude, latitude, scale});
```

Comment:
Should this be `[long, lat]`.

Reply:
`getWorldPosition` takes named parameters